pkg/services/ngalert/notifier/alertmanager.go

```diff
@@ -308,6 +309,155 @@ func (am *Alertmanager) SyncAndApplyConfigFromDatabase() error {
 	return nil
 }
 
+func (am *Alertmanager) TestReceivers(ctx context.Context, c apimodels.TestReceiversConfig) (*apimodels.TestReceiversResult, error) {
```

Comment:
I'm fine with this being part of the  `Alertmanager` public interface but can we move this to a separate file? I find that this file is getting a bit too big and testing of receivers is not really an "Alertmanager" concept is more of a "Grafana" concept.